rita_common/src/debt_keeper/mod.rs

```diff
@@ -337,10 +336,15 @@ fn deserialize_from_json(file_path: String) -> Option<DebtDataSer> {
     }
 }
 
+pub fn get_pay_thresh() -> Int256 {
+    let payment = settings::get_rita_common().payment;
```

Comment:
this function and the function for calculating the close threshold should probably be in the same file, also comment needed here.

Reply:
done